pallets/schema/src/mock.rs

```diff
@@ -49,8 +54,8 @@ where
 	}
 }
 
-pub fn generate_schema_id<T: Config>(digest: &SchemaHashOf<T>) -> IdentifierOf {
-	let identifier: IdentifierOf = curi::generate(&(&digest).encode()[..], SCHEMA_PREFIX)
+pub fn generate_schema_id<T: Config>(digest: &SchemaHashOf<T>) -> SchemaIdOf {
+	let identifier = Ss58Identifier::generate(&(&digest).encode()[..])
```

Comment:
generate is not part of URI anymore. Use Ss58Identifier::to_schema_id()

Reply:
Done